lisp/ess-r-package.el

```diff
@@ -310,9 +338,8 @@ filter argument. Otherwise, use the whole base filename."
 })
 ")
 
-(defun ess-r-devtools-revdep-check-package (&optional alt)
+(defun ess-r-devtools-revdep-check-package ()
```

Comment:
Should you remove `interactive` here?

Maybe we should retire this function altogether, I have never used it and I think it's outdated. I wouldn't be surprised if no one uses it.

Reply:
I haven't used it either and I also don't think it's useful. revdep is a complex procedure which almost always would require a separate script file. So, yes .. let's retire.